service/resource/cloudformation/spec.go

```diff
@@ -1,5 +1,10 @@
 package cloudformation
 
+const (
+	// defaultCreationTimeout is the timeout in minutes for the creation of the stack
```

Comment:
Godoc is represented by full sentences. The full stop is missing.

Reply:
Done